account_invoice_supplier_ref_unique/views/res_config_settings.xml

```diff
@@ -9,30 +9,16 @@
         <field name="model">res.config.settings</field>
         <field name="inherit_id" ref="account.res_config_settings_view_form" />
         <field name="arch" type="xml">
-            <xpath
-                expr="//div[@name='show_purchase_receipts_right_pane']/parent::div"
-                position="after"
-            >
-                <div
-                    class="col-xs-12 col-md-6 o_setting_box"
-                    title="If you check this box, the system verifies that supplier invoice number is unique across the system."
+            <xpath expr="//block[@id='account_vendor_bills']" position="inside">
+                <setting
+                    id="show_purchase_receipts"
```

Comment:
the id should be check_invoice_supplier_number, isn't it?

Reply:
Right, Updated.

Thanks